### ISSUE TITLE
Allow for multiple sites

### ISSUE THREAD

**Issue by AaronDovTurkel** [ShailenNaidoo/Netlify#16]
HI,
I often deploy to 3 different testing servers.
Can you add functionality so that all servers can be set in configs?

Thanks!

**Comment by ShailenNaidoo** [comment]
@AaronDovTurkel Hey :wave: This sounds like a great idea, can you maybe elaborate on the details, please?

**Comment by jochemkeller** [comment]
Hi! Passing through after Googling, thought I'd leave my solution.

You can add a unique `netlify.site_id` property to `.vscode/settings.json` in the root of each of your VS Code project workspaces. This way, each project workspace has it's own Netlify site linked to it 👍

**Comment by ShailenNaidoo** [comment]
@jochemkeller Yeah that makes, the details of the issue was a little obscure so I thought @AaronDovTurkel was referring to have 3 Netlify observers in one workspace. Thanks helping out :)

**Comment by AaronDovTurkel** [comment]
@ShailenNaidoo sorry for the obscurity. I actually was referring to having three observers in the same workspace... I have three servers building from different branch deploys and the catch is I cant push up the vscode settings to bitbucket. (@jochemkeller thanks for your help). Any suggestions?

-> for extra clarity, I would like the ability to see three different builds at the same time! (I don't know if there's an easier solution like having an extension that just puts Netlify build badges in the status bar...)

**Comment by ShailenNaidoo** [comment]
@AaronDovTurkel Yeah I guess that is possible but not sure if I'll have the time to implement that. Let me put a word out to the community and see if it is something people are actually keen on. Ill let you know :)

**Comment by AaronDovTurkel** [comment]
@ShailenNaidoo in the meantime [here's](https://marketplace.visualstudio.com/items?itemName=TheHolyCoder.badge-viewer) a bare-bones extension I just wrote for viewing badges...

**Comment by ShailenNaidoo** [comment]
@AaronDovTurkel Are you perhaps able to do a PR for this?

**Comment by AaronDovTurkel** [comment]
ill see if I have some downtime and check out the code base!

**Comment by teddydaniels** [comment]
@AaronDovTurkel Just following up on whether you are still keen on doing a PR for this?
